File: 2868/Errrou.py
def solve():
    eq = input().split()
    result = 0
    if eq[1] == "+":
        result = int(eq[0]) + int(eq[2])
    if eq[1] == "-":
        result = int(eq[0]) - int(eq[2])
    if eq[1] == "x":
        result = int(eq[0]) * int(eq[2])

    
    qtde = abs(result - int(eq[4]))
    
    print("E"+qtde*"r"+"ou!")


t = int(input())

for _ in range(t):
    solve()

File: 2868/Errrou.cpp
#include <bits/stdc++.h>
#include <stdlib.h>

void solve(){
    int a, b, result, sum;
    char signal, equal;

    std::cin >> a >> signal >> b >> equal >> result;

    if (signal == '+') 
        sum = a + b;
    else if (signal == '-')
        sum = a - b;
    else
        sum = a * b;

   

    int digits = abs(sum - result);

    std::cout << "E" << std::string(digits,'r') << "ou!" << std::endl;

}



int main(){
    std::ios::sync_with_stdio(false);
    std::cin.tie(nullptr);


    

    int t;

    std::cin >> t;

    while (t--){
        solve();
    }


}
